modules/nodes-nodes-jobs-creating-cron.adoc

```diff
@@ -21,43 +21,45 @@ kind: CronJob
 metadata:
   name: pi
 spec:
-  schedule: "*/1 * * * *"  <1>
-  concurrencyPolicy: "Replace" <2>
-  startingDeadlineSeconds: 200 <3>
-  suspend: true            <4>
-  successfulJobsHistoryLimit: 3 <5>
-  failedJobsHistoryLimit: 1     <6>
-  jobTemplate:             <7>
+  schedule: "*/1 * * * *"          <1>
+  timeZone: Etc/UTC                <2>
+  concurrencyPolicy: "Replace"     <3>
+  startingDeadlineSeconds: 200     <4>
+  suspend: true                    <5>
+  successfulJobsHistoryLimit: 3    <6>
+  failedJobsHistoryLimit: 1        <7>
+  jobTemplate:                     <8>
     spec:
       template:
         metadata:
-          labels:          <8>
+          labels:                  <9>
             parent: "cronjobpi"
         spec:
           containers:
           - name: pi
             image: perl
             command: ["perl",  "-Mbignum=bpi", "-wle", "print bpi(2000)"]
-          restartPolicy: OnFailure <9>
+          restartPolicy: OnFailure <10>
 ----
 +
 <1> Schedule for the job specified in link:https://en.wikipedia.org/wiki/Cron[cron format]. In this example, the job will run every minute.
-<2> An optional concurrency policy, specifying how to treat concurrent jobs within a cron job. Only one of the following concurrent policies may be specified. If not specified, this defaults to allowing concurrent executions.
+<2> An optional time zone for the schedule. See link:https://en.wikipedia.org/wiki/List_of_tz_database_time_zones[List of tz database time zones] for valid options. If not specified, the Kubernetes controller manager interprets the schedule relative to its local time zone. This setting is offered as a link:https://access.redhat.com/support/offerings/techpreview[Technology Preview].
+<3> An optional concurrency policy, specifying how to treat concurrent jobs within a cron job. Only one of the following concurrent policies may be specified. If not specified, this defaults to allowing concurrent executions.
 * `Allow` allows cron jobs to run concurrently.
 * `Forbid` forbids concurrent runs, skipping the next run if the previous has not
 finished yet.
 * `Replace` cancels the currently running job and replaces
 it with a new one.
-<3> An optional deadline (in seconds) for starting the job if it misses its
+<4> An optional deadline (in seconds) for starting the job if it misses its
```

Comment:
You could consider changing the parenthesis to commas:

> <4> An optional deadline, in seconds, for starting the job if it misses its scheduled time. Missed job executions will be counted as failed. If not specified, there is no deadline.

In that example, I also changed "jobs executions" to "job executions". Additionally, I removed "for any reason" and "ones" to minimise the sentence.